addons/mail/views/res_config_settings_views.xml

```diff
@@ -77,6 +77,18 @@
                         <setting id="restrict_template_rendering_setting" help="Restrict mail templates edition and QWEB placeholders usage.">
                             <field name="restrict_template_rendering"/>
                         </setting>
+                        <setting id="tenor_url" string="Tenor API url" help="">
+                            <field name="tenor_url"/>
+                        </setting>
+                        <setting id="tenor_api_key" string="Tenor API key" help="Add a tenor api key to enable gifs support." documentation="https://developers.google.com/tenor/guides/quickstart#setup">
+                            <field name="tenor_api_key" placeholder="Paste your API key"/>
+                        </setting>
+                        <setting id="tenor_content_filter" string="Tenor content filter" help="Select the content filter used for filtering gifs" documentation="https://developers.google.com/tenor/guides/content-filtering">
+                            <field name="tenor_content_filter"/>
+                        </setting>
+                        <setting id="tenor_gif_limit" string="Tenor gif limits" help="Fetch up to the specified number of gif.">
+                            <field name="tenor_gif_limit"/>
+                        </setting>
```

Comment:
Instead of just mentioning the service as `Tenor`, I would use `Tenor GIF` (so it's very clear it's about Gifs).

- Tenor GIF API URL
- Tenor Gif API Key (Add a Tenor GIF api key to enable gifs support.)
- Tenor GIF Content Filter
- Tenor GIF Limits

Also prefer`GIFs` instead of `gifs`, as this is an acronym